source/localizable/tutorial/autocomplete-component.md

```diff
@@ -102,6 +102,7 @@ Now, define your new controller like so:
 ```app/controllers/index.js
 export default Ember.Controller.extend({
   filteredList: null,
+  model: null,
```

Comment:
But running the script without this line, makes the code throwing error when the code reaches this line:
this.set('model').clear(); on line 21
The error thrown:
> index.js:25 Uncaught TypeError: Cannot read property 'clear' of undefined

The error occures when you search for empty string.

Clone my repo and test it:
[https://github.com/twify93/emberjs-rentals](url)